tools/c7n_tencentcloud/c7n_tencentcloud/provider.py

```diff
@@ -42,7 +42,7 @@ def initialize(self, options):
             options.region = options.regions[0]
         if not options.account_id:
             session = self.get_session_factory(options)
-            options.account_id = session.client(
+            options.account_id = session().client(
```

Comment:
fwiw the reason this is behind a factory abstraction is when we start. to deal with sts/cross account assumption aspects, but whether that maps directly in Tencent cloud is a little unclear, but that the sts api does exist does imply it to me.